examples/Demo/Shared/Pages/Icon/IconPage.razor

```diff
@@ -65,6 +58,56 @@
     </Description>
 </DemoSection>
 
+<h2>Customization</h2>
+
+<p>
+    You can create your own icons by adding a class like this one:
+
+    <CodeSnippet Language="language-csharp">public static class MyIcons
+{
+    public class SettingsEmail : Icon { public SettingsEmail() : base("SettingsEmail", IconVariant.Regular, IconSize.Size20, "&lt;svg width=\"20\" height=\"19\" viewBox=\"0 0 20 19\" fill=\"none\" xmlns=\"http://www.w3.org/2000/svg\">&lt;path d=\"M15.6251 2.5H4.37508L4.2214 2.50428C2.79712 2.58396 1.66675 3.76414 1.66675 5.20833V13.125L1.67103 13.2787C1.75071 14.7029 2.93089 15.8333 4.37508 15.8333H9.76425C9.91725 15.4818 10.1354 15.1606 10.4087 14.8873L10.7126 14.5833H4.37508L4.25547 14.5785C3.50601 14.5177 2.91675 13.8902 2.91675 13.125V6.97833L9.709 10.5531L9.78908 10.5883C9.95267 10.647 10.135 10.6353 10.2912 10.5531L17.0834 6.9775V9.17258C17.5072 9.14483 17.9362 9.21517 18.3334 9.38358V5.20833L18.3292 5.05465C18.2494 3.63038 17.0693 2.5 15.6251 2.5ZM4.37508 3.75H15.6251L15.7447 3.75483C16.4942 3.81568 17.0834 4.44319 17.0834 5.20833V5.565L10.0001 9.29375L2.91675 5.56583V5.20833L2.92158 5.08873C2.98242 4.33926 3.60994 3.75 4.37508 3.75ZM15.9167 10.5579L10.9979 15.4766C10.7112 15.7633 10.5077 16.1227 10.4093 16.5162L10.0279 18.0418C9.86208 18.7052 10.4631 19.3062 11.1265 19.1403L12.6521 18.7588C13.0455 18.6605 13.4048 18.4571 13.6917 18.1703L18.6103 13.2516C19.3542 12.5078 19.3542 11.3018 18.6103 10.5579C17.8665 9.814 16.6605 9.814 15.9167 10.5579Z\" fill=\"#212121\" />&lt;/svg>") { } }
+}    </CodeSnippet>
+</p>
+
+
+<p>
+    If the size of your customized icon (Viewbox) is not one of the standard IconSize sizes,
+    you can use a <b>IconSize.Custom</b> size.
+    <br />
+    <ul>
+        <li>
+            <FluentIcon Value="@(new MyCircle())" Width="16px" /> 
+            <code>&lt;FluentIcon Value="@@(new MyCircle())" Width="16px" /></code>
+        </li>
+        <li>
+            <FluentIcon Value="@(new MyCircle().WithColor("red"))" Width="16px" /> 
+            <code>&lt;FluentIcon Value="@@(new MyCircle().WithColor("red"))" Width="16px" /></code>
+        </li>
+        <li>
+            @(new MyCircle().ToMarkup("16px", "blue")) 
+            <code>@@new MyCircle().ToMarkup("16px", "blue")</code>
+        </li>
+        <li>
+            <FluentButton IconStart="@(new MyCircle())" Loading="@ButtonIsLoading" OnClick="@(e => ButtonIsLoading = !ButtonIsLoading)">Refresh</FluentButton>
+            <code>&lt;FluentButton IconStart="@@(new MyCircle())">Refresh&lt;/FluentButton></code>
+        </li>
+    </ul>
```

Comment:
Are you showing IconSize.Custom being used here because of using the `Width` parameter?